pkg/operator/client/client.go

```diff
@@ -10,8 +10,7 @@ import (
 	routev1 "github.com/openshift/api/route/v1"
 	gatewayapiv1 "sigs.k8s.io/gateway-api/apis/v1"
 
-	maistrav1 "github.com/maistra/istio-operator/pkg/apis/maistra/v1"
-	maistrav2 "github.com/maistra/istio-operator/pkg/apis/maistra/v2"
+	sailv1 "github.com/istio-ecosystem/sail-operator/api/v1"
```

Comment:
Does this need update to match up with the servicemesh3 operator?

Reply:
I don't think so.  Since OSSM and the upstream Sail Operator use the same GVK, there shouldn't be any breaking changes.  I see some minor differences between them, mostly around kubebuilder validation (which doesn't matter when we're just importing the Go definitions) and some API fields that we aren't using anyway.